tools/code.js

```diff
@@ -162,8 +162,8 @@ function processVariables(content) {
             variables.split(/\r\n|\r|\n/gu).map(function (line) {
```

Comment:
Please remove the `u` flag because of errors in minified build :)

Reply:
This is in a tool, not in sources.

Reply:
Im to sensitive after last problems :D